microcosm_pubsub/dispatcher.py

```diff
@@ -76,34 +83,27 @@ def handle_message(self, media_type, message):
                 )
                 return handler_with_context(message)
             except SkipMessage as skipped:
-                self.logger.info(
+                logger.info(
```

Comment:
I would probably merge unless we add a base pubsub handler that automatically merges the context into extra. No way everyone raising the exception will remember to put the message context in there on their own.